dask/array/fft.py

```diff
@@ -73,9 +92,17 @@ def _ihfft_out_chunks(a, n, axis):
 
 
 _out_chunk_fns = {'fft': _fft_out_chunks,
+                  'fft2': _fft_out_chunks,
+                  'fftn': _fft_out_chunks,
                   'ifft': _fft_out_chunks,
+                  'ifft2': _fft_out_chunks,
+                  'ifftn': _fft_out_chunks,
                   'rfft': _rfft_out_chunks,
+                  'rfft2': _rfft_out_chunks,
+                  'rfftn': _rfft_out_chunks,
                   'irfft': _irfft_out_chunks,
+                  'irfft2': _irfft_out_chunks,
+                  'irfftn': _irfft_out_chunks,
                   'hfft': _hfft_out_chunks,
                   'ihfft': _ihfft_out_chunks}
```

Comment:
No, I don't think that they will change.  My comment is trying to expose the structure that the choice of chunk function depends only on the prefix to fft, not the suffix.